content/en/blog/2023/end-user-discussions-03.md

```diff
@@ -0,0 +1,273 @@
+---
+title: OpenTelemetry End-User Discussions Summary for March 2023
+linkTitle: End-User Discussions Mar 2023
+date: 2023-03-30
+author: '[Reese Lee](https://github.com/reese-lee) (New Relic)'
+body_class: otel-with-contributions-from
+---
+
+With contributions from [Henrik Rexed](https://github.com/henrikrexed)
+(Dynatrace), [Michael Hausenblas](https://github.com/mhausenblas) (AWS),
+[Rynn Mancuso](https://github.com/musingvirtual) (Honeycomb),
+[Adriana Villela](https://github.com/avillela) (Lightstep), and
+[Pranay Prateek](https://github.com/pranay01) (SigNoz).
+
+The OpenTelemetry end-user group meet takes place every month for users in the
+Americas (AMER), Europe Middle-East & Africa (EMEA), and Asia-Pacific (APAC).
+
+The discussions take place using a
+[Lean Coffee format](https://agilecoffee.com/leancoffee/), whereby folks are
+invited to post their topics to the
+[Agile Coffee board like this one](http://agile.coffee/#3716060f-183a-4966-8da4-60daab2842c4),
+and everyone in attendance votes on what they want to talk about.
+
+## What we talked about
+
+Sampling and collector capabilities continue to be topics of interest, along
+with questions about instrumentation and adoption.
+
+## Discussion Highlights
+
+Below is the summary of this month's discussions.
+
+### OpenTelemetry Collector
+
+#### 1 - Losing gRPC with Azure App Services
+
+**Q:** When looking at the hosting models in Azure for the OTel Collector, only
+HTTP is supported (for running in Azure App Service). What are the risks
+associated with losing gRPC capability?
+
+**A:** If HTTP2 is supported in Azure, gRPC might work there, since gRPC is HTTP
+under the hood with extra complications built on top of HTTP2. One suggestion is
+to follow up with Microsoft about gRPC support, as it may have very long-running
+connections.
+
+#### 2 - Uptime monitoring/synthetics
+
+**Q:** Does the OTel Collector have the capability to do uptime monitoring/
+synthetics? If not, are there any plans to work towards such a thing?
+
+**A:** The
+[health check](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/healthcheckextension/README.md)
+might be a helpful reference. Also check out the
+[http check receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/healthcheckextension/README.md).
+
+#### 3 - Collector distributions
+
+**Q:** Should I use a vendor distribution versus the community collector
+distribution?
+
+**A:** Each vendor distribution will come with customizations, whereas the
+community Collector distribution will include everything: receivers and
+exporters. If you need the flexibility, then you should use the OTel Collector
+distro.
+
+#### 4 - Rate limiting on receivers
+
+**Q:** Each vendor distribution will come with customizations, whereas the
```

Comment:
I think this is a duplication of the previous question.

Reply:
Ooo bad copy-paste job on my part (I started the post in a Google doc). Thanks for catching this!